SimonImanigirimpuhwe/my-brand-website
# #7 ft(contact-page): add contact page

#### What does this PR do?

- Add Contact page for inqueries

#### Description of Task to be completed?

-Add contact information

- Add navigation links to the page

- Add social media links

- Add form for sending inquiries

#### How should this be manually tested?

- Navigate to the page you will see contact information and form for sending inquiries

#### Any background context you want to provide?

- N/A

#### What are the relevant Trello stories?

- [#7](https://trello.com/c/YMotw8TJ/7-contact-page)

#### Screenshots (if appropriate)


#### Questions:

- N/A
